src/Bicep.Core.Samples/Files/InvalidLambdas_LF/main.diagnostics.bicep

```diff
@@ -34,9 +34,10 @@ var filter4 = filter(range(0, 10), () => null)
 //@[35:45) [BCP070 (Error)] Argument of type "() => null" is not assignable to parameter of type "any => bool". (CodeDescription: none) |() => null|
 var filter5 = filter(range(0, 10), i => i)
 //@[04:11) [no-unused-vars (Warning)] Variable "filter5" is declared but never used. (CodeDescription: bicep core(https://aka.ms/bicep/linter/no-unused-vars)) |filter5|
-//@[35:41) [BCP070 (Error)] Argument of type "int => int" is not assignable to parameter of type "any => bool". (CodeDescription: none) |i => i|
+//@[35:41) [BCP070 (Error)] Argument of type "(0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9) => (0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9)" is not assignable to parameter of type "any => bool". (CodeDescription: none) |i => i|
```

Comment:
Do we need some sort of cutoff in how we display the type to the user if this list gets very long? (e.g. what happens if the above is `range(0, 100)` instead of `range(0, 100)`).

Reply:
I think you’ll have a similar question when you get to the LargeTemplate_Stress_LF symbols file update.

I’m not sure what the right approach here would be. Tuple names seem more useful for small arrays and much less useful for empty or large ones.

Reply:
I couldn't come up with a good use case for having range return a tuple, so I've reverted that signature update back to `int[]`.